srcpkgs/llvm/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'llvm'
 pkgname=llvm
```

Comment:
imo just have llvm8 provides="llvm-8.0.0_${revision}"

Reply:
Hum, didn't we say on LLVM6->7 that provides is rather messy and that a metapackage is nicer to use?